vizro-core/docs/pages/user-guides/run.md

```diff
@@ -4,13 +4,16 @@ This guide shows you how to launch your dashboard in different ways. By default,
 
 ## Py.Cafe
 
-You can launch your dashboard and edit the code live on [Py.Cafe](https://py.cafe/).
+The easiest way to launch your dashboard is to edit the code live on [Py.Cafe](https://py.cafe/).
 
-For this basic example, follow the [Edit live on Py.Cafe](https://py.cafe/stichbury/vizro-iris-analysis) link below the dashboard to experiment with the code.
+Most of our examples have a link below the code — [Run and edit this code in Py.Cafe](https://py.cafe/vizro./vizro-iris-analysis-0) — which you can follow to experiment with the code. This will lead you to an editor such as the one below, which displays the dashboard and the code side by side.
 
-<iframe src="https://py.cafe/stichbury/vizro-iris-viz" width="100%" height="600px"></iframe>
+<div style="width: 100%; height: 400px;">
+    <iframe src="https://py.cafe/vizro./vizro-iris-analysis-0?theme=light&linkToApp=True" style="border: 1px solid #e6e6e6; border-radius: 8px; max-width: unset; width: calc(100%/0.5); height: calc(400px/0.5); transform-origin: top left; transform: scale(0.5);" ></iframe>
+</div>
```

Comment:
Is this supposed to be interactive so e.g. do we want people to be able to edit code from here? If yes, I think it's too blurry and zoomed out for any useful interaction. 

I would replace it with a high-resolution GIF instead if you just wanted to demo the editor mode?

Reply:
Good point, I replaced it with a screenshot in the end. I think that works best!